### PR TITLE
Switching to glide vendoring tool.

### PR DESCRIPTION
**Pull request by GaiusMartius**
To build enter 'glide install' then 'go build'

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -0,0 +1,12 @@
+hash: 791aaf9a24179856621317e12f08cf40a722596868353ecaa13f190a18daa7c3
```
Comment (adtac) [inline glide.lock]: Could you add a small note on the development section of README explaining how to generate this automatically? (it it automatic, right?)

**Comment by adtac** [comment]
Oh and could you please use a shorter commit shortlog? (50 or fewer chars)

You could write the bulk of your message in the commit body (and please limit that to 72 chars too)

File: glide.lock (hunk 1 above)
Comment (GaiusMartius) [inline glide.lock]: Sure

**Comment by adtac** [comment]
ack https://github.com/adtac/commento/pull/91/commits/7aec1bfbc478d3435110b71a30dea2bcf8c25683 https://github.com/adtac/commento/pull/91/commits/8b517c01d9bfbadad0a3963af117cca0b7eaa774 https://github.com/adtac/commento/pull/91/commits/5cedd38632e5ea18d9ce9729b56c44f4d3592239

**Comment by adtac** [comment]
requires a rebase: `git pull --rebase origin master`

**Comment by cweagans** [comment]
Would it be worth using `dep` instead now? That seems to be the direction that everything is heading.

**Comment by GaiusMartius** [comment]
@cweagans, That's up to @adtac obviously but I will say I brought up using `dep` previously. The consensus was that it should be considered stable first before it's used in the app.

It's fortunate then when the time to switch comes it will involve little more than
`dep init`
`dep ensure`

**Comment by cweagans** [comment]
I saw some of that discussion. That was around March of this year though, and there's been a lot of progress since then. Just thought it'd be worth revisiting based on technical progress + community buy-in.

**Comment by adtac** [comment]
I'll evaluate all major go dependency management tools later tonight and see which is the best one for us :+1:

**Comment by adtac** [comment]
@GaiusMartius @cweagans (CC @DarkThrone as he was involved in discussions earlier)

I think we'll go with dep from now on. It looks like dep is expected to be relatively stable after go1.9 and since we're past that, it's a good choice now. The last time we reviewed the tool was too long ago, but if this is the direction the community is heading towards, and if us choosing to use dep will add momentum towards solving the mess that is Go's dependency management, let's go with it.

@GaiusMartius I have incorporated your other PR on adding dep into my PR; lots of things changed since that PR, so this was easier.

@cweagans @GaiusMartius it'd be great if you could take a look at master's `Gopkg.{toml,lock}` and see if everything is alright. The `.lock` file is generated. I've fixed the dependencies on the latest stable release. (Which reminds me, I need to release a Commento version.)

Closing this PR as we're not going to be using Glide anymore. Closing that other PR adding `dep` docs to README. That has been incorporated into `docs/development.md`.

**Pull request closed without merging**